tests/system/testdata/connections/list.txtar

```diff
@@ -50,8 +50,8 @@ return code == 0
 output equals ''
 
 ak connection list --integration meow -J
-return code == 1
-output equals 'Error: integration "meow" not found'
+return code == 0
```

Comment:
This is good, but please also add a test of the previous error when using `--fail`, for completeness.

Reply:
added